benchmark/index.html

```diff
@@ -50,7 +50,9 @@ <h1>WebNN Benchmark</h1>
             <h4>Model</h4>
             <select id="modelName">
               <option value="mobilenet_v1_tflite" selected>Mobilenet v1 (TFLite)</option>
+              <option value="mobilenet_v1_quant_tflite" selected>Mobilenet v1 Quant (TFLite)</option>
```

Comment:
why `selected` here?

Reply:
@Wenzhao-Xiang thanks, I have submitted another patch to fix this issue.